backend/requirements-dev.txt

```diff
@@ -1,8 +1,5 @@
-
-responses
```

Comment:
I was confused as I thought we were still using this, and indeed we are but it was listed twice in the file :P